feed.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0">
+  <channel>
+    <title>People and Code, At Your Disposal</title>
+    <link>https://people-and-code.com/</link>
+    <description>Latest articles from People and Code</description>
+    <lastBuildDate>Tue, 04 Mar 2025 11:24:36 GMT</lastBuildDate>
+    <item>
```

Comment:
![image](https://github.com/user-attachments/assets/ff748478-8050-465e-95f4-34ead0ef3e70)
Extra elements needed: 
- An illustration (even if it's only our logo)
- A description field with a summary or the first paragraph and a link to the rest of the article.,

Reply:
Now supports a single image. Additional support requires the pages themselves to have an associated image. 

One way to do this is by using an og:image meta tag e.g.

`<meta name="image" property="og:image" content="https://inaturalist-open-data.s3.amazonaws.com/photos/97465767/medium.jpeg?1601194277">`